kubejs/server_scripts/base/featrues/devices.js

```diff
@@ -72,6 +72,40 @@ if (
         removeRecipe({ id: "create:crafting/kinetics/controller_rail" });
       },
     },
+    {
+      tag: "forge:device/furniture",
+      generic: "ptdye:furnished_device",
+      base: "minecraft:stick",
+      incomplete: "ptdye:incomplete_furnished_device",
```

Comment:
Not needed. As long as it's using the same format it will find the incomplete version. Only needed for items that don't have incomplete versions